File: Examples/波浪/NYWaterWave-master/NYWaterWaveDemo/ViewController.m
//
//  ViewController.m
//  NYWaterWaveDemo
//
//  Created by 牛严 on 16/8/16.
//  Copyright © 2016年 牛严. All rights reserved.
//

#import "ViewController.h"
#import "NYWaterWaveView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [self setUp];
}

- (void)setUp
{
    NYWaterWaveView *waterWaveView = [[NYWaterWaveView alloc]initWithFrame:CGRectMake(0, 200, self.view.frame.size.width, 200)];
    [self.view addSubview:waterWaveView];
}

@end

File: WaterWave/WaterWave/ViewController.swift
//
//  ViewController.swift
//  WaterWave
//
//  Created by 伯驹 黄 on 16/9/29.
//  Copyright © 2016年 xiAo_Ju. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let v = WaterWaveView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: 200))
        view.addSubview(v)
    }
    
    override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

